Sources/DronecodeSDKSwiftDemo/Controllers/MapViewController.swift

```diff
@@ -44,22 +45,23 @@ class MapViewController: UIViewController, CLLocationManagerDelegate {
     }
 
     private func initButtonStyle() {
-        uploadMissionButton.layer.cornerRadius   = UI_CORNER_RADIUS_BUTTONS
-        startMissionButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        createFlightPathButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        centerMapOnDroneButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        pauseMissionButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        setCurrentIndexButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        downloadMissionButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
-        isMissionFinishedButton.layer.cornerRadius    = UI_CORNER_RADIUS_BUTTONS
+        uploadMissionButton.layer.cornerRadius   		= UI_CORNER_RADIUS_BUTTONS
+        startMissionButton.layer.cornerRadius    		= UI_CORNER_RADIUS_BUTTONS
+		cancelMissionUploadButton.layer.cornerRadius 	= UI_CORNER_RADIUS_BUTTONS
```

Comment:
This looks odd on GitHub. Do you use the same spaces/tabs settings as the existing project?

Reply:
This was something I did to clean up the code. It can be corrected.